lib/ransack/context.rb

```diff
@@ -135,7 +135,7 @@ def ransackable_association?(str, klass)
     end
 
     def ransackable_scope?(str, klass)
-      klass.ransackable_scopes(auth_object).any? { |s| s.to_s == str }
+      klass.ransackable_scopes(auth_object).any? { |s| s.to_sym == str.to_sym }
```

Comment:
We can assume that most people use symbols instead of strings, so this prevents 13 KB of string allocations without sacrificing backward compatibility.

Reply:
ref #857 CC @seanfcarroll